articles/tech-resources/custom-bounce-domain.md

```diff
@@ -16,37 +16,36 @@ To configure a bounce domain, choose the subdomain you would like to use (e.g. m
 
 This DNS record will ensure that out-of-band/asynchronous bounces are routed to SparkPost for processing.
 
-Once you have configured your DNS settings, register and verify the domain with  SparkPost by following the steps below:
+Once you have configured your DNS settings, register and verify the domain with SparkPost by following the steps below:
 
 1. Register the domain with SparkPost by using the sending domains [create endpoint](https://developers.sparkpost.com/api/sending-domains.html#sending-domains-create-post)
-1. CNAME-verify the domain by using the sending domains [verify endpoint](https://developers.sparkpost.com/api/sending-domains.html#sending-domains-verify-post).  Here's an example API call to CNAME-verify the mail.example.com domain:
+2. CNAME-verify the domain by using the sending domains [verify endpoint](https://developers.sparkpost.com/api/sending-domains.html#sending-domains-verify-post).  Here's an example API call to CNAME-verify the mail.example.com domain:
 
     ```
-    POST /api/v1/sending-domains/mail.example.com/verify
+    POST /api/v1/sending-domains/mail.yourdomain.com/verify
 
     {
       "cname_verify" : true
     }
     ```
 
-1. Optionally set the domain as your account default bounce domain so that it is automatically used as the bounce domain for all messages sent through the account.  Here's an example API call to set mail.example.com as the account default bounce domain:
+3. Optionally set the domain as your account default bounce domain so that it is automatically used as the bounce domain for all messages sent through the account.  Here's an example API call to set mail.example.com as the account default bounce domain:
 
     ```
-    PUT /api/v1/sending-domains/mail.example.com
+    PUT /api/v1/sending-domains/mail.yourdomain.com
```

Comment:
We conventionally used to use `example.com` in our sample code and documentation. Any reason for this change?